models/lung_nodule_ct_detection/scripts/cocometric_ignite.py

```diff
@@ -0,0 +1,112 @@
+from typing import Callable, Dict, Sequence, Union
+
+import torch
+from ignite.metrics.metric import Metric, reinit__is_reduced, sync_all_reduce
+from monai.apps.detection.metrics.coco import COCOMetric
+from monai.apps.detection.metrics.matching import matching_batch
+from monai.data import box_utils
+
+from .utils import detach_to_numpy
+
+
+class IgniteCocoMetric(Metric):
```

Comment:
Is it possible to inherit this `IgniteMetric` class to simplify the logic?
https://github.com/Project-MONAI/MONAI/blob/dev/monai/handlers/ignite_metric.py#L32
I think you can refer to the `DiceMetrc`:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/handlers/mean_dice.py#L19
And actually, I feel maybe you can move this handler to MONAI apps directly?
CC @wyli What do you think?

Thanks.

Reply:
Inheriting from IgniteMetric cannot simply it...